tests/test_core.py

```diff
@@ -8456,6 +8456,19 @@ def test():
     self.emcc_args += ['-DPOOL']
     test()
 
+  def test_errno_type(self):
```

Comment:
I think this should be in `other`, as we don't need to automatically test it in each optimization level. Just a single test is enough i think?

Reply:
Makes sense; done.